src/FormAlter/MediaFormAlter.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+declare(strict_types = 1);
```

Comment:
Interesting, this is the first time I've seen this. TIL. What led you to add this here? Is this something your adding to all new classes these days?

Reply:
@m4olivei I've been adding it whenever I remembered for the last few months. Just last week I actually added to my file templates in phpStorm. 

whenever that's declared in a file, it'll keep php from doing the automatic casting it usually does when, for example, you declare a typehint on a method argument, but then the user pass another type. By default, php would cast it. With this, it'll throw an error, so I'm basically enforcing that whoever is calling methods, is passing the correct data from the calling place, and doing there whatever casting is needed, explicitly.